cypress/e2e/hub/collections.cy.ts

```diff
@@ -222,9 +222,23 @@ describe('Collections Details View', () => {
 
   it.skip('user can deprecate a collection', () => {});
 
-  it.skip('user can delete version from system', () => {});
+  it('user can delete version from system', () => {
+    cy.approveCollection(collection, namespace, '1.0.0');
+    cy.galaxykit('task wait all');
+    cy.get('[data-cy="table-view"]').click();
+    cy.clickLink(collection);
```

Comment:
I would suggest to use cy.filterTableBySingleText(collection) before you call cy.clickLink(collection) that will filter for this particular rows, we have now single texts filter (that not allows for filter by multiple text keys) and this broke some of the tests.